externals/kyuubi-spark-sql-engine/pom.xml

```diff
@@ -167,6 +167,7 @@
                     <shadedArtifactAttached>false</shadedArtifactAttached>
                     <artifactSet>
                         <includes>
+                            <include>com.alibaba:druid</include>
```

Comment:
HiKariCP should the better choice, evaluated from popularity, performance, and has native support of dropwizard and prometheus.  
https://github.com/apache/dolphinscheduler/issues/6881

Reply:
i think in this situation, Druid can meet our requirements.
even though HiKariCP is springboot‘s default pool.

Reply:
> even though HiKariCP is springboot‘s default pool.

It's doesn't matter, the metrics is matter, both kyuubi and spring use dropwizard, and HiKariCP already in Spark runtime classpath

Reply:
We can list the reason why we choose `Druid`, otherwise the `HiKariCP` seems enough.

Reply:
ok, wait

Reply:
done
use Hikari